middleware/cmd/middleware/main.go

```diff
@@ -18,19 +18,19 @@ import (
 const version string = "0.0.1"
 
 func main() {
-	middlewarePort := flag.String("middlewareport", "8845", "Port the middleware should listen on (default 8845)")
-	electrsRPCPort := flag.String("electrsport", "51002", "Electrs rpc port")
-	dataDir := flag.String("datadir", ".base", "Directory where middleware persistent data like noise keys is stored")
-	network := flag.String("network", "testnet", "Indicate wether running bitcoin on testnet or mainnet")
-	bbbConfigScript := flag.String("bbbconfigscript", "/opt/shift/scripts/bbb-config.sh", "Path to the bbb-config file that allows setting system configuration")
-	bbbCmdScript := flag.String("bbbcmdscript", "/opt/shift/scripts/bbb-cmd.sh", "Path to the bbb-cmd file that allows executing system commands")
-	bbbSystemctlScript := flag.String("bbbsystemctlscript", "/opt/shift/scripts/bbb-systemctl.sh", "Path to the bbb-systemctl script that allows starting and stopping services on the Base.")
-	prometheusURL := flag.String("prometheusurl", "http://localhost:9090", "Url of the prometheus server in the form of 'http://localhost:9090'")
+	middlewarePort := flag.String("middlewareport", "8845", "Port the Middleware listens on")
+	electrsRPCPort := flag.String("electrsport", "51002", "Electrs RPC port")
+	dataDir := flag.String("datadir", ".base", "Directory where the Middleware persistent data, like for example the noise encryption keys, is stored")
+	network := flag.String("network", "testnet", "Indicate wether Bitcoin is running on mainnet or testnet")
```

Comment:
Not sure why we default to `testnet` here. Maybe @TheCharlatan knows more?

Reply:
Because I was lazy and always running testnet in the beginning. Feel free to change to mainnet.